srt-tokio/src/multiplex.rs

```diff
@@ -1,9 +1,16 @@
-use std::{collections::HashMap, io, net::SocketAddr, time::Instant};
+use std::{
```

Comment:
We can probably remove multiplex now that we have SrtListener, which does the same thing, right?

Reply:
Yes, that was the intent.